homeassistant/components/schluter/climate.py

```diff
@@ -0,0 +1,165 @@
+"""Support for Schluter thermostats."""
+from datetime import timedelta
+import logging
+
+from requests import RequestException
+import voluptuous as vol
+
+from homeassistant.components.climate import PLATFORM_SCHEMA, ClimateDevice
+from homeassistant.components.climate.const import (
+    CURRENT_HVAC_HEAT,
+    CURRENT_HVAC_IDLE,
+    HVAC_MODE_HEAT,
+    SUPPORT_TARGET_TEMPERATURE,
+)
+from homeassistant.const import ATTR_TEMPERATURE, CONF_SCAN_INTERVAL
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from . import DATA_SCHLUTER_API, DATA_SCHLUTER_SESSION, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+SCAN_INTERVAL = timedelta(seconds=5)
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
The default platform schema that climate uses already contains a scan interval. No need to define it.

Reply:
SCAN_INTERVAL is now pulled from the default platform schema 👍

Reply:
if you do that, it means that you don't need to define it below. It looks like you can remove all `PLATFORM_SCHEMA` related constant.

Reply:
Please remove the `PLATFORM_SCHEMA` completely.